components/badge/index.css

```
@@ -146,7 +148,7 @@
 /* windows high contrast mode */
 @media (forced-colors: active) {
 	.spectrum-Badge {
-		border-color: CanvasText; /* stylelint-disable-line declaration-property-value-no-unknown */
+		border-color:  var(--highcontrast-spectrum-badge-border-color);
```

Giving a quick look at existing `--highcontrast-` custom properties, it looks like "spectrum" is usually dropped, like:

```suggestion
		border-color:  var(--highcontrast-badge-border-color);
```

---

Nice catch! Just updated that! :)